app/src/main/java/com/example/next_show/data/ShowFilterer.java

```diff
@@ -0,0 +1,40 @@
+package com.example.next_show.data;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+public class ShowFilterer {
+    // instance vars
+    private HashMap<String, List<String>> filters;
+
+    // genres for shows
+    private static final String ACTION = "action";
+    private static final String COMEDY = "comedy";
+    private static final String DRAMA = "drama";
+    private static final String ALL_SHOWS = "all";
+
+    // filters
+    public static final String GENRE = "genre";
+    public static final String NETWORK = "network";
+    public static final String YEAR = "year";
+
+    public ShowFilterer () {
+
+    }
+
+    private HashMap<String, List<String>> setUpFilters() {
```

Comment:
It looks like this isn't called yet, anywhere?

Reply:
Also, do you need this method?  Would selecting everything not be the same as having no filters?